channels/do.m3u

```diff
@@ -25,60 +23,42 @@ http://177.52.221.214:8000/play/a0f7/index.m3u8
-https://video.misistemareseller.com/Fuegotv/Fuegotv/chunklist_w917415169.m3u8
+https://video.misistemareseller.com/Fuegotv/Fuegotv/playlist.m3u8
 #EXTINF:-1 tvg-id="GDMTELEVISION.do" tvg-name="GDM TELEVISION" tvg-country="DO" tvg-language="" tvg-logo="https://i0.wp.com/www.gdm.do/wp-content/uploads/2020/01/Logo-GDM.png?w=696&ssl=1" group-title="",GDM TELEVISION
-https://ss2.domint.net:3200/gdm_str/gdmtv/chunklist_w1317041912.m3u8
+https://ss2.domint.net:3200/gdm_str/gdmtv/playlist.m3u8
 #EXTINF:-1 tvg-id="GHTELEVISIONCANAL10HD.do" tvg-name="GH TELEVISION CANAL 10 HD" tvg-country="DO" tvg-language="" tvg-logo="" group-title="",GH TELEVISION CANAL 10 HD
 https://cloudflare.streamgato.us:3352/live/ghtelevisionhdlive.m3u8
-#EXTINF:-1 tvg-id="GHTELEVISIONCANAL10HD.do" tvg-name="GH TELEVISION CANAL 10 HD" tvg-country="DO" tvg-language="" tvg-logo="https://yt3.ggpht.com/ytc/AAUvwninYlGRCnZuK3DTrVqyzK4bbB9LoZUSJI-pww3yRA=s900-c-k-c0x00ffffff-no-rj" group-title="",GH TELEVISION CANAL 10 HD
-https://cloudflare.streamgato.us:3352/live/ghtelevisionhdlive.m3u8?DVR
 #EXTINF:-1 tvg-id="HermanasMirabalTV.do" tvg-name="Hermanas Mirabal TV" tvg-country="DO" tvg-language="" tvg-logo="https://emisoradominicana.net/television/wp-content/uploads/2015/06/hermanas-mirabal-tv.jpg" group-title="",Hermanas Mirabal TV
-https://cm.hostlagarto.com:4445/Hmtv/hmtv.myStream/chunks.m3u8
+https://cm.hostlagarto.com:4445/Hmtv/hmtv.myStream/playlist.m3u8
 #EXTINF:-1 tvg-id="HilandoFino.do" tvg-name="Hilando Fino" tvg-country="DO" tvg-language="Spanish" tvg-logo="http://www.hilandofinotv.com/wp-content/uploads/2017/04/cropped-logo-tv-3.png" group-title="",Hilando Fino (720p)
 https://4k-server-mia.2cdn.eu//HilandoFinoTV/ngrp:HilandoFinoTV_all/playlist.m3u8
 #EXTINF:-1 tvg-id="Hits360TV.do" tvg-name="Hits 360 TV" tvg-country="DO" tvg-language="" tvg-logo="https://i.imgur.com/Cx5V233.jpg" group-title="Music",Hits 360 TV
 http://cm.hostlagarto.com:8081/hits360tv/hits360HD.myStream/playlist.m3u8
 #EXTINF:-1 tvg-id="HM.do" tvg-name="HM" tvg-country="DO" tvg-language="Spanish" tvg-logo="http://emisoradominicana.net/television/wp-content/uploads/2015/06/hermanas-mirabal-tv.jpg" group-title="",HM (720p)
 http://cm.hostlagarto.com:8081/Hmtv/hmtv.myStream/playlist.m3u8
 #EXTINF:-1 tvg-id="LAMIATV.do" tvg-name="LA MIA TV" tvg-country="DO" tvg-language="Spanish" tvg-logo="https://i.imgur.com/77uVMGw.png" group-title="",LA MIA TV (720p)
-https://ss8.domint.net:3108/mia_str/lamiatv/chunklist_w2025663703.m3u8
+https://ss8.domint.net:3108/mia_str/lamiatv/playlist.m3u8
 #EXTINF:-1 tvg-id="LocomotionTV.do" tvg-name="Locomotion TV" tvg-country="LATAM;PT;ES" tvg-language="Spanish" tvg-logo="https://i.imgur.com/auEJJvP.png" group-title="",Locomotion TV (480p)
 http://51.222.85.85:81/hls/loco/index.m3u8
 #EXTINF:-1 tvg-id="LocomotionTV.do" tvg-name="Locomotion TV" tvg-country="DO" tvg-language="Spanish" tvg-logo="https://i.imgur.com/auEJJvP.png" group-title="",Locomotion TV
```

Comment:
Animation channel as well as AnimeZoneTV

Reply:
Adding it. I don't often look at tags when it comes to links and stuff.